modules/microshift-cni-multus-nad-bridge-nw-config.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-nad-bridge-nw-config_{context}"]
+= Creating a NetworkAttachmentDefinition for the bridge additional network
+
+Use the following procedure to create a `NetworkAttachmentDefinition` configuration file for a bridge-type additional network. You can also use the example workflow here that uses `host-local` IP address management (IPAM) to configure other supported additional network types.
+
+[IMPORTANT]
+====
+If you use `bridge` and the `dhcp` IPAM, a DHCP server listening on the bridged network is required. If you are also using a firewall, configuring the firewalld service to allow DHCP traffic on the network zone is also required. You can run the `firewall-cmd --remove-service=dhcp` command in this case.
+====
+
+.Prerequisites
+
+* The {microshift-short} Multus CNI is installed.
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+
+.Procedure
+
+. Optional: Verify that the {microshift-short} cluster is running with the Multus CNI by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-multus
+----
++
+.Example output
+[source,terminal]
+----
+NAME                READY   STATUS    RESTARTS   AGE
+dhcp-daemon-dfbzw   1/1     Running   0          5h
+multus-rz8xc        1/1     Running   0          5h
+----
+
+. Create a `NetworkAttachmentDefinition` configuration file by running the following command and using the following example file for reference:
++
+[source,terminal]
+----
+$ oc apply -f network-attachment-definition.yaml
+----
++
+.Example `NetworkAttachmentDefinition` file
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-conf
+spec:
+  config: '{
+      "cniVersion": "0.4.0",
+      "type": "bridge", <1>
+      "bridge": "br-test", <2>
+      "mode": "bridge",
+      "ipam": {
+        "type": "host-local", <3>
+        "ranges": [
+          [
```

Comment:
We support dual stack for pod's secondary interface,  You can add IPv6 addresses referenced by https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-72796

Reply:
Although IPv6 should work with the additional network. I'd suggest we wait to include IPv6 examples in the docs until the full MicroShift cluster can support IPv6, with 4.17.